config/serverless.yml

```diff
@@ -9,6 +9,7 @@ xpack.fleet.internal.activeAgentsSoftLimit: 25000
 
 # Cloud links
 xpack.cloud.base_url: "https://cloud.elastic.co"
+xpack.cloud.projects_url: "/projects/"
```

Comment:
Curious what are your thoughts on this? 

We will propagate the correct value from cloud, but I'd like to have something displayed locally for testing. Is it fine to keep it here? Or should I hardcode some fallback from code?

Reply:
I wouldn't commit it in the repo. Good to mention it in the PR description for local testing. I removed all those URLs here https://github.com/elastic/kibana/pull/161971/files#diff-ebeb9e9dc4c3ee5e81f743f0eda3c06f1fc161a11127b374d21f0da5e6368fd9

Reply:
So I am not sure about this, because developers starting it locally would not see the cloud URLs you've added in the sidenav and wouldn't see this link in the header. 

@elastic/kibana-core, Curious, what would you recommend?

Reply:
Seems that we need a `serverless.dev.yml` then

Reply:
One alternative is to handle this at the config level:

```
  projects_url: schema.conditional(
      schema.contextRef('serverless'),
      true,
      schema.string({ defaultValue: '/projects'}),
      schema.maybe(schema.string())
  )
```

Would that be appropriate for something like this setting? This way it can always be overridden, but `--serverless` flag would make it default to `/projects`.

> Seems that we need a serverless.dev.yml then

Feel like this might be a broader discussion as it would be useful for cases other than this!

Reply:
> One alternative is to handle this at the config level:

I like this approach, we also have other URLs like this that we didn't add to config, so they are undefined locally and aren't visible. I'd personally prefer to see them even if they lead to nowhere or to a wrong env. 
https://github.com/elastic/kibana/pull/161971/files#diff-ebeb9e9dc4c3ee5e81f743f0eda3c06f1fc161a11127b374d21f0da5e6368fd9

Reply:
@jloleysens, I've follow you suggestion for this new config: https://github.com/elastic/kibana/pull/163076/commits/eebd9fb89535cdb84534817dd42619a0ba24c898

I didn't touch other existing configs

Reply:
I am not sure our future selves want to start putting configs in multiple places. I would inject all the URL configs from a single place (cloud) and have a way to set default in a local `yml` file.